webserver/pkgpkr/webservice/recommender_service.py

```diff
@@ -71,12 +75,35 @@ def get_recommendations(self, dependencies):
         recommended = []
         for result in cur.fetchall():
 
-            # Format metadata
+            # Package name
             package = result[0].replace("pkg:npm/", "", 1)
+
+            # Recommendation name
             recommendation = result[1].replace("pkg:npm/", "", 1)
+
+            # Recommendation URL
             url = f"{NPM_DEPENDENCY_BASE_URL}/{self.name_only_regex.search(result[1]).group(1)}"
-            similarity = math.ceil(10 * result[2])
-            day = result[5]
+
+            # Similarity score
+            similarity_score = math.ceil(10 * result[5])
+
+            # Popularity score
+            popularity_score = result[4]
```

Comment:
accessing by index is not optimal, but should be fine, otherwise, we could throw it into a named tuple or a dict or a class to access by attribute then. probably no much time for optimizations now

Reply:
Agree. Next time we have to touch this code, let's refactor it to avoid indexing.